components/AppToolbar.vue

```diff
@@ -1,12 +1,28 @@
+<script setup type="ts">
+const route = useRoute();
+const showBackButton = () =>  route.name !== "index" && route.name !== "search" && route.name !== "browse";
```

Comment:
We can turn this into a `computed` for it to be updated reactively:)

Reply:
done